### PR TITLE
Creates layout and fragment for ADS1115 and MLX90614

### PR DESCRIPTION
**Pull request by asitava1998**
Fixes issue #340 

Changes: 
* Created layout files for `ADS1115` and `MLX90614`.
Changes remaining:
* Create fragment files for the above

Screenshots for the change: 
`MLX90614`

`ADS1115`

The fragments work well but there is an issue with the values displayed by every sensor. I don't think the problem is sensor specific or in the sensor files but with something else.

### CODE REVIEW

**Comment by viveksb007** [comment]
@asitava1998 Please resolve conflicts and ```Configure``` cardView there is lot of space left after spinner and spinner text is cut off. Could you adjust that part so that spinner text is properly readable.
Thanks

**Comment by asitava1998** [comment]
@viveksb007 Merge conflicts fixed and increased width of the spinners. :+1:

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -187,9 +197,38 @@
...
+    <string-array name="ads1115_rate_spinner">
+        <item>8</item>
+        <item>16</item>
+        <item>32</item>
+        <item>64</item>
+        <item>128</item>
+        <item>250</item>
+        <item>475</item>
+        <item>860</item>
+
```
Comment (viveksb007) [inline app/src/main/res/values/strings.xml]: end string array by tag ```</string-array>```. And Please check the build once before pushing the commit.

Comment (asitava1998) [inline app/src/main/res/values/strings.xml]: That got deleted most probably while resolving conflicts. Usually, I resolve conflicts directly from github as it is easier that way.

**Pull request merged**